### PR TITLE
Add better error for email invites without identity server

### PR DESCRIPTION
**Pull request by t3chguy**
Fixes https://github.com/vector-im/element-web/issues/16893

---
Here's what your changelog entry will look like:

## ✨ Features
 * Add better error for email invites without identity server ([\#10739](https://github.com/matrix-org/matrix-react-sdk/pull/10739)). Fixes vector-im/element-web#16893.

### CODE REVIEW

**Review by kerryarchibald — APPROVED** [review]
Unit test coverage?

File: src/utils/MultiInviter.ts [hunk 1]
```diff
@@ -277,6 +277,12 @@ export default class MultiInviter {
                                 errorText = _t("The user's homeserver does not support the version of the room.");
                             }
                             break;
+                        case "ORG.MATRIX.JSSDK_MISSING_PARAM":
```
Comment (richvdh) [inline src/utils/MultiInviter.ts:280]: are you sure this is the only reason this error can be returned?

File: src/utils/MultiInviter.ts [hunk 2]
```diff
@@ -277,6 +277,12 @@ export default class MultiInviter {
...
+                            if (getAddressType(address) === AddressType.Email) {
+                                errorText = _t(
+                                    "Cannot invite user by email without an identity server, connect to one in settings",
```
Comment (richvdh) [inline src/utils/MultiInviter.ts:283]: Comma splice

```suggestion
                                    'Cannot invite user by email without an identity server. You can connect to one under "Settings".',
```

File: src/utils/MultiInviter.ts (hunk 1 above)
Comment (t3chguy) [reply src/utils/MultiInviter.ts:280]: Currently yes, especially down the 3pid/email codepaths.

**Pull request merged**